src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -51,17 +53,25 @@ private void InitializeDefaultAutoCompleteCandidates()
         
         internal void PopulateAutoCompleteCandidates()
         {
+            searchElementsCache = new List<NodeSearchElement>();
```

Comment:
I don't think we need to initialize it here as it is being assigned on line 60 anyway.

Reply:
Since we are storing the search results in a list, when we call the PopulateAutoCompleteCandidates API, we want to clear the previous matching node elements and update it with the new list.

Reply:
Won't the previous cache be cleared when you point this cache to a new list?

Reply:
Yes, I missed that part. Removing this initialization.